### DISCUSSION TITLE
Ps2 reset using keycombo from controller

### DISCUSSION THREAD

**Discussion by AceCuba** [darthcloud/BlueRetro#742] [General]
is posible Ps2 system reset using keycombo from controller , using a internal esr32 mod and a bt controller?

**Comment by Invictaz** [comment]
Yes but need to have additional solder points on the main PS2 PCB, see project Methylene from Gusse on Twitter.

**Comment by YgorSan** [comment]
it's a very interesting and useful idea, anyway for those who use homebrews on console like xeb+, launchelf, retroarch, etc alternatively you can do it for them, or if you use the igr feature present in opl and popstarter you can run another one homebrew with a predefined combination of buttons after starting a game, in addition to shutting down the console as well.
